arsenal/sleep/aws/serverless.yml

```diff
@@ -112,6 +162,51 @@ functions:
     handler: com.serverless.Handler
     memorySize: 1536
 
+  java11-memory-128:
+    package:
+      artifact: java11/target/hello-dev.jar
```

Comment:
@nwb-adir it's ok that this directory doesn't exist?

Reply:
fixed this